### ISSUE TITLE
Upgrade postcss and chart.js dependencies to avoid security vulnerabilities

### ISSUE THREAD

**Issue by jessex** [Recidiviz/supervision-success-component#59]
**What needs to be done? Why does it need to be done?**
There are two fresh, active dependency vulnerabilities for this repo: postcss and chartjs. The latter is a high severity vulnerability so we should resolve these within the next 7 days.

**Additional context**
[Vulnerabilities listed here](https://github.com/Recidiviz/supervision-success-component/security/dependabot)

**Comment by macfarlandian** [comment]
auto-generated PRs #60 and #61 have been merged and there are no more reported vulnerabilities. 

The postcss one seems to have been another phantom alert, dependabot said our version was no longer vulnerable and then the alert just vanished after I merged #61 ... meanwhile #60 never seemed to have an alert associated with it on this repo, although the vulnerability is real and I'm seeing it on some of our other repos? tough week for our friend dependabot. 

anyway since chart.js is part of the client application bundle, we should disseminate a new build of this application now to complete the resolution of this vulnerability.
